cli/src/main/kotlin/io/spine/protodata/cli/Main.kt

```diff
@@ -84,122 +82,155 @@ private fun readVersion(): String {
  * The main CLI command which performs the ProtoData code generation tasks.
  *
  * The command accepts class names for the service provider interface implementations via the CLI
- * parameters, such as `--plugin`, `--renderer`, `--option-provider`, and `--options`, all of which
- * can be repeated parameters, if required. Then, using the classpath of the app and
- * the user classpath supplied via the `--user-classpath` parameter, loads those classes.
+ * parameters, such as `--plugin`, `--renderer`, and `--option-provider`, all of which
+ * can be repeated parameters, if required.
+ *
+ * Then, using the classpath of the app and the user classpath supplied via the `--user-classpath`
+ * parameter, loads those classes.
+ *
  * `Code Generation` context accept Protobuf compiler events, regarding the Protobuf types, listed
- * in the `CodeGeneratorRequest.file_to_generate` as loaded from the `--request` parameter. Finally,
- * the renderers apply required changes to the source set with the root path, supplied in
+ * in the `CodeGeneratorRequest.file_to_generate` as loaded from the `--request` parameter.
+ *
+ * Finally, the renderers apply required changes to the source set with the root path, supplied in
  * the `--source-root` parameter.
  */
 internal class Run(version: String) : CliktCommand(
     name = "protodata",
-    help = "ProtoData tool helps build better multi-platform code generation. Version ${version}.",
+    help = "ProtoData tool helps build better multi-platform code generation." +
+            System.lineSeparator() +
+            "Version ${version}.",
     epilog = "https://github.com/SpineEventEngine/ProtoData/",
     printHelpOnEmptyArgs = true
 ) {
+    /** Option names that are used in the help texts of other options. */
+    private object Op {
+        const val RENDERER = "--renderer"
+        const val SOURCE_ROOT = "--source-root"
+    }
+
+    /** Configuration option names that are used in the help texts of other options. */
+    private object ConfigOpt {
+        const val FILE = "--configuration-file"
+        const val VALUE = "--configuration-value"
+        const val FORMAT = "--configuration-format"
+    }
+
+    /** Abbreviation extension. */
+    private fun String.ti() = trimIndent()
```

Comment:
It'd be great to check if the compiler treats this thing the same.

I suspect that the Kotlin compiler applies string manipulations such as `trimIndent()` and `trimMargin()` on literals at compile time. I wonder if it's smart enough to do that with `ti()`.

This is not a call for action, just a point of interest.

Reply:
Assuming that `trimIndent()` itself is done this way...
```kotlin
public fun String.trimIndent(): String = replaceIndent("")
```
I don't think we're at risk.